src/metabase/util.cljc

```diff
@@ -1145,3 +1145,11 @@
          (let [acc1 (reduce f init r1)
                acc2 (reduce f acc1 r2)]
            acc2)))))
+
+(defn run-count!
```

Comment:
I ended up not using this, but I could seem myself needing it in future.

Will remove if anyone is against just keeping it.